README.md

```diff
@@ -216,6 +237,30 @@ const User =
     })
 ```
 
+### ID Fields
+
+It is possible to declare a field as an ID. This metadata will be used by glues to enrich the infrastructure interfaces (Database, REST, GraphQL, etc).
+
+We can do it in two ways:
+
+```javascript
+// The explicit way
+const User =
+    entity('User', {
+        id: field(Number, { isId: true }),
+        ...
+    })
+
+// The short way
+const User =
+    entity('User', {
+        id: id(Number),
+        ...
+    })
+```
+
+ID Fields only accepts ```Number``` or ```String``` type.
```

Comment:
How a id typed as `Date` is a problem? It might not look right, but I can't see why we should constraint the type. It is a developer decision that won't impact in any way Herbs architecture.

Reply:
I remove this in the documentation.